group_vars/derrida_crawl_qa/vars.yml

```diff
@@ -12,6 +12,10 @@ browsertrix_crawl_repo_branch: staging
 browsertrix_collection: "derrida"
 browsertrix_custom_driver: "derrida-driver.js"
 
+# Set at 5 hours to ensure that the custom driver can finish hovering over all
+#  visualization elements
+browsertrix_timeout_seconds: 18000
```

Comment:
Great! Please document this as an optional parameter in the readme for the browsertrix role

Reply:
✅ d197c4f5de50bd29c3729a7afa88f104a4134991